### ISSUE TITLE
How to display a note in horizontal mode?

### ISSUE THREAD

**Issue by Exkywor** [Darakah/obsidian-timelines#65]
First of all, amazing plugin! It's exactly what I was looking for.
However, I was testing it out today, and I saw in the video that in the horizontal mode if you click on an event you can get it to display the image and then jump to the note with the event itself, similar to what is displayed in the vertical mode, but on my end I can only see the range box, but nothing else about it.

This is what I get:
![image](https://github.com/Darakah/obsidian-timelines/assets/46603136/037feb5b-31d7-406b-9242-76ee7b57ad80)

No matter where I click on the event, I can't get it to display its info or jump to its note.

**Comment by ReconVirus** [comment]
is note preview on in your core plugin of obsidian?

**Comment by Exkywor** [comment]
> is note preview on in your core plugin of obsidian?

Yup. If I Ctrl+Click the event a glitch happens for a second where the thumbnail is displayed inside the event name box, but it disappears instantly and nothing happens.

**Comment by ReconVirus** [comment]
you are referring to something like this right? 
![image](https://github.com/Darakah/obsidian-timelines/assets/43733760/f76a0cbd-1b88-4218-8e3a-825c506ac342)

**Comment by Exkywor** [comment]
> you are referring to something like this right? ![image](https://user-images.githubusercontent.com/43733760/264217628-f76a0cbd-1b88-4218-8e3a-825c506ac342.png)

I think so?
This is from the video:
![image](https://github.com/Darakah/obsidian-timelines/assets/46603136/c56b8971-9b1c-4db3-ad28-4f08ff630718)

**Comment by Exkywor** [comment]
> best advise would be to stop using this as the code hasnt be maintained, and obsidian has changed alot since. you can look at https://github.com/seanlowe/obsidian-timelines
> 
> but in you really want to keep with it, try a reinstall, that seems to fix whatever problem...temporarily

Didn't know about the fork, thanks!
